run_seas5.py

```diff
@@ -0,0 +1,27 @@
+import sys
+import tempfile
+
+from constants import BBOX_STR_GLOBAL, BBOX_STR_TEST
+from seas5.download_archive_seas5_tprate import *
+
+
+if __name__ == "__main__":
+
+    if len(sys.argv) < 4:
+        print("Usage: python run_mars.py <scope> <start_year> <end_year>")
+        sys.exit(1)
+
+    args = sys.argv
+    scope = args[1]
+    start_year = int(args[2])
+    end_year = int(args[3])
```

Comment:
think we should add some additional assertions/limits on all the arguments, `scope` , `start_year`, `end_year`

here is what I've tried so far and the output which is not super clear to me:

```
python run_seas5.py test 2024 2024
#> Running SEAS5 update from 2024 to 2024 with test scope
```

```
python run_seas5.py test 2023 2023
#> Running SEAS5 update from 2023 to 2023 with test scope
```

Did runs work? 2024 shouldn't really work as it's not on MARS yet? Also what I expect should be happening is that this test run should have downloaded both sets of `.grib` files and `COG` files to a tempdir. However, how do i access/inspect that this is true?

Perhaps we should add a `dir` argument where the either both `.grib` and `.tif` or just `.tif` files can be dropped for inspection?

Reply:
Great points. I've switched to using the `argparse` module, which lets us do the assertions pretty easily. I've limited the "archive" download to 2022. I've also added a `--test` flag which will download the raw and processed data locally to a `test_outputs/` directory.